testing/test_dataset.py

```diff
@@ -13,11 +13,12 @@
 import sys
 import unittest
 import mock
-
+import endaq.ide as ei
```

Comment:
`endaq-python` is a different library that has `idelib` as a requirement. It really shouldn't be used in `idelib` tests.